### PR TITLE
ALP zSeries deployment

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: tasks/deploy-alp-zseries.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+          Link the TCPMAINT disk to have the FTP command available:
+        </para>
+<screen>VMLINK TCPMAINT 592</screen>
+      </step>
+      <step>
+        <para>
+          Connect to the FTP server and download the required files for IPLing
+          installation. In our case, the anonymous user is allowed:
+        </para>
+<screen>FTP example.org (addr ipv4
```
Comment (imobachgs) [inline tasks/deploy-alp-zseries.xml:139]: Is the closing parenthesis missing? It is missing from the source document, so I guess it is fine. @teclator, please, could you check?

**Review by imobachgs — APPROVED** [review]
It looks good, thanks a lot! I only have a question for @teclator about a missing parenthesis. But nothing to block this PR.

File: tasks/deploy-alp-zseries.xml (hunk 1 above)
Comment (teclator) [reply tasks/deploy-alp-zseries.xml:139]: It is correct

**Comment by dariavladykina** [comment]
Hi Tomas, just letting you know that I cannot make suggestions to the existing text after this PR has been merged 🙁
Could we open a new branch for it maybe? I've run through it and spotted some typos I'd like to fix.

Thanks!
D.
________________________________

**Pull request merged**